src/components/Verses/BookmarksSection.tsx

```diff
@@ -0,0 +1,37 @@
+import useTranslation from 'next-translate/useTranslation';
+
+import BookmarkedVersesList from './BookmarkedVersesList';
+import styles from './BookmarksAndQuickLinks.module.scss';
+
+import Tabs from 'src/components/dls/Tabs/Tabs';
+
+enum View {
+  Bookmarks = 'bookmarks',
+  Popular = 'popular',
+}
+
+const BookmarksSection = () => {
+  const { t } = useTranslation('home');
+
+  const tabs = [{ title: t('tab.bookmarks'), value: View.Bookmarks }];
+
+  return (
+    <div>
+      <div className={styles.tabsContainer}>
+        <Tabs
+          tabs={tabs}
+          selected={View.Bookmarks}
+          onSelect={() => {
```

Comment:
I would rather make this optional inside `Tabs` and not pass it at all and inside `Tabs` dls, we can do this:

```
          {...(onSelect && {
            onKeyDown: () => onSelect(tab.value),
            onClick: () => onSelect(tab.value),
          })}
```